src/js/modules/utilities/Logging.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+import * as winston from "winston";
+
+// TODO: Note if chosen to use this, be careful as the file transports are
+// asynchronous and winston doesn't support a flush() call. Flush() must be
+// called before the program exists in success & failure cases. For now
+// just use console logging
+export function newFileLogger(
```

Comment:
This is unused and untested code, up to maintainers weather to keep it or not

Reply:
remove?, keep it on a local branch ?

Reply:
@andresaristizabal if you want to take a peek

Reply:
Going to remove for now